common/utils.py

```diff
@@ -23,3 +23,10 @@ def render_confirmation_email(user: CustomUser, domain: str) -> str:
             "token": account_activation_token.make_token(user),
         },
     )
+
+
+def render_create_user_email(user: CustomUser, domain: str, password: str) -> str:
```

Comment:
There is missing unit test for this function

Reply:
Added test for passing proper parameters.